docs/docs/start-here.md

```diff
@@ -92,6 +92,10 @@ Since we [released version 1.0.0](/blog/release-1.0.0) we've seen some positive
 reaction in the community. Here are some interesting posts, articles, and
 videos:
 
+*   The November 2025 release of Spring Framework 7.0 was
+    [announced](https://spring.io/blog/2024/10/01/from-spring-framework-6-2-to-7-0),
+    and it promises that its "null-safety strategy is converging with the recently released JSpecify annotations".
```

Comment:
Maybe "mentions" or "states" instead of "promises"? If you want to share a stronger commitment, you can also link [my comment](https://github.com/spring-projects/spring-framework/issues/28797#issuecomment-2387137015) in the JSpecify issue which is targeting Spring Framework 7.

Reply:
That sounds good. Thanks!